docs/api/argtypes.md

```diff
@@ -85,6 +85,12 @@ These values--description, table.type, and controls.type--get merged over the de
 
 In particular, this would render a row with a modified description, a type display with a dropdown that shows the detail, and no control.
 
+<div class="aside">
+
+NOTE: Just as it happens with other properties (such as `args`), `argTypes` can be overridden in a single story.
```

Comment:
@alfupe we can simplify it a bit we could go with

"As it happens with other properties (such as `args`), `argTypes` can be overridden in a single story." 

Shorter, simpler and leaner, what do you think?

Reply:
@jonniebigodes I agree, it sounds more natural like that, thanks a lot!